components/Conversation/RecipientControl.tsx

```diff
@@ -122,7 +125,7 @@ const RecipientControl = ({
           <AddressInput
             recipientWalletAddress={recipientWalletAddress}
             id="recipient-field"
-            className="block w-[95%] pl-7 pr-3 pt-[3px] md:pt-[2px] md:pt-[1px] bg-transparent caret-n-600 text-n-600 placeholder-gray-500 focus:outline-none focus:placeholder-gray-400 focus:ring-0 focus:border-transparent text-lg font-mono"
```

Comment:
Maybe these styling changes snuck into the branch. They don't feel related to the ultimate goal of the PR.